src/styles/blocks/container.scss

```diff
@@ -0,0 +1,20 @@
+@import "../utils/variables";
+@import "../utils/functions";
```

Comment:
You've imported these files in each block file. Can't understand why you did do so?

Let's import them once in the `index.scss` file